### DISCUSSION TITLE
Google Analytics script should be in <head>

### DISCUSSION THREAD

**Discussion by nunomazer** [razonyang/hugo-theme-bootstrap#925] [General]
According to [Analytics Help Page](https://support.google.com/analytics/answer/9304153#zippy=%2Cadd-the-google-tag-directly-to-your-web-pages) the tag script should be inserted just ahter <head> tag.  

![image](https://user-images.githubusercontent.com/652935/230739106-e8da5a1f-1b33-497a-a977-d620e20542d5.png)

**Comment by razonyang** [comment]
Fixed by #926, included in tag `v1.0.0-beta5.6`.
Thanks for your feedback.
